src/memory/ArrayProxy.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <memory/IAnyArray.hpp>
+
+struct ArrayHub
```

Comment:
This class seems to be not used anywhere and should probably be moved out of this request.